runelite-client/src/main/java/net/runelite/client/plugins/pvptools/PvpToolsPlugin.java

```diff
@@ -293,6 +295,7 @@ private void addSubscriptions()
 		eventBus.subscribe(GameStateChanged.class, this, this::onGameStateChanged);
 		eventBus.subscribe(PlayerSpawned.class, this, this::onPlayerSpawned);
 		eventBus.subscribe(PlayerDespawned.class, this, this::onPlayerDespawned);
+		eventBus.subscribe(MenuEntryAdded.class, this, this::onMenuEntryAdded);
```

Comment:
When are you unsubscribing? You should subscribe when the config changes (wouldn't even have to cache the field) and unsubscribe when it turns false. You could use a string with something like "pvptools_menuentries" as the lifecycle, so you don't have to unregister all events.

Only subscribing when needed (ie when you don't need the if statements in front of the method) is something we can all improve on imo